src/audio/drc/drc_math_hifi3.c

```diff
@@ -38,19 +38,7 @@
 #define INV_FUNC_A2_Q25 1126492160 /* Q_CONVERT_FLOAT(33.57208251953125f, 25) */
 #define INV_FUNC_A1_Q25 -713042175 /* Q_CONVERT_FLOAT(-21.25031280517578125f, 25) */
 #define INV_FUNC_A0_Q25 239989712 /* Q_CONVERT_FLOAT(7.152250766754150390625f, 25) */
-
-/*
- * Input depends on precision_x
- * Output range [0.5, 1); regulated to Q2.30
- */
-static inline ae_f32 rexp_fixed(ae_f32 x, int32_t precision_x, int32_t *e)
-{
-	int32_t bit = 31 - AE_NSAZ32_L(x);
-
-	*e = bit - precision_x;
-
-	return AE_SRAA32(x, bit - 30);
-}
+#define TWENTY_Q26 1342177280 /*20 << 26*/
```

Comment:
align with above definition.

Reply:
which one do you like?

Reply:
#define TWENTY_Q26 1342177280 /*  Q_CONVERT_FLOAT(20, 26) */